components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -1530,6 +1531,29 @@ def getMaxPlaneSize(self):
                 int(c.getConfigValue('omero.pixeldata.max_plane_height')))
         return self._maxPlaneSize
 
+    def getOrphanedImagesSettings(self):
+        default = {
+            "enabled": True,
+            "name": "Orphaned Images",
+            "description": ("This is a virtual container with "
+                            "orphaned images.")
+        }
```

Comment:
@will-moore test improvement is about to keep state of the default config to prevent unexpected failures. We never used it in gateway as you well know. Without better way of retrieving config from the server (with defaults) I don't see this being improvement. This will be extra call.